### DISCUSSION TITLE
Config questions

### DISCUSSION THREAD

**Discussion by xucaen** [dosemu2/dosemu2#1638] [General]
I'm sorry for asking such simple questions but I have spent hours searching the web for answers.

1) I followed directions on here (https://github.com/dosemu2/dosemu2/issues/1145) to copy my ~/.dosemurc but I can't get dosemu to use it. How do I tell dosemu to use my local config file? 

2) when I start dosemu, it shows me a C:> prompt, which I found is being mapped to ~/.dosemu/drive_c. How do I change that to be ~/dos  ?

**Comment by stsp** [comment]
Please use `~/.dosemu/.dosemurc`.
No need to copy anything (don't
trust docs :), just see comments of
the setting `$_hdimage` and put
only that setting to your local config
file.

**Reply by stsp** [reply]
Also please look up the
`$_hostfs_drives` option.
All options are well documented
in the default config file itself.

**Comment by xucaen** [comment]
$_hostfs_drives looks like what I want to use, but I still can not get my
.dosemurc to be read.

I created ~/.dosemu/.dosemurc
and it contains this line:

$_hostfs_drives = "~/dos"

I crested ~/dos directory and left it empty, but when I run dosemu, it
still uses ~/.dosemu/drive_c as the C: drive

I googled "$_hostfs_drives" but not finding an answer. Any suggestion are
appreciated.
Thanks!

**Comment by stsp** [comment]
$_hostfs_drives does not change your
drive C. But it can add some other drive.
You can use --Fdrive_c to change C.

**Comment by xucaen** [comment]
Your missing the primary issue. Dosemu refuses to use my config file. How
to do I fix it?

**Reply by stsp** [reply]
How do you know it ignores
your config file? The only "test"
you mentioned was that you
put `$_hostfs_drives = "~/dos"`
and expected your drive C to
change, but it won't.
Put `qwerty` into your config
file (or any other bad line),
and if it is really ignored,
then dosemu2 will start
nevertheless.

**Comment by xucaen** [comment]
Because I also set com1 to vmodem and telix doesn't see the com port. But
when I change the system wide config it does.

I'm on my phone so I can't get you exact paths right now. Hopefully you
know what I mean by 'system wide  config"

I'm good with change that one if the local one won't work, I just would
like to keep everything local.

**Reply by stsp** [reply]
Local one shold work, and
works for everyone.
Please attach boot.log and
maybe it will give a clue.

**Comment by xucaen** [comment]
It occurs to me you meant the dosemu boot.log. Attaching that now.

**Reply by stsp** [reply]
And nothing is attached...

**Comment by xucaen** [comment]
That's strange, I wonder if something on your end filtered it out. I still
see it attached to the email I'll try zipping it and attaching it again.
[image: image.png]

**Reply by stsp** [reply]
I am not emailing with you.
Please get to the github page
and attach the file here.
